newIDE/app/src/MainFrame/Onboarding/OnboardingDialog.js

```diff
@@ -0,0 +1,109 @@
+// @flow
+import { Trans } from '@lingui/macro';
+
+import * as React from 'react';
+import userflow from 'userflow.js';
+import Dialog from '../../UI/Dialog';
+import FlatButton from '../../UI/FlatButton';
+import optionalRequire from '../../Utils/OptionalRequire';
+import { getProgramOpeningCount } from '../../Utils/Analytics/LocalStats';
+import { isMobile } from '../../Utils/Platform';
+import Window from '../../Utils/Window';
+import { loadPreferencesFromLocalStorage } from '../Preferences/PreferencesProvider';
+import { getUserUUID } from '../../Utils/Analytics/UserUUID';
+import RaisedButton from '../../UI/RaisedButton';
+import { Column, Line } from '../../UI/Grid';
+import { MarkdownText } from '../../UI/MarkdownText';
+
+const electron = optionalRequire('electron');
+const isDev = Window.isDev();
+export let isUserflowRunning = false;
+
+const onboardingText = `
+In 5 minutes, you will have:
+  - Created a game
+  - Learned the fundamentals of GDevelop
+
+(🇬🇧 The tour is only available in English)  
+We highly recommend it!
+`;
+
+const OnboardingDialog = () => {
+  const [open, setOpen] = React.useState(false);
+  const [loading, setLoading] = React.useState(false);
+
+  // Initialise userflow when modal opens.
+  React.useEffect(() => {
+    if (isDev) {
```

Comment:
This will activate UserFlow for 100% of the people seeing this dialog. Instead, let's init userflow only when the tour is really started (`startUserflow` method).

Reply:
I wasn't sure which method was considered as a new user for userfow.
But let's be safe and not do anything until we're sure we want to show the flow to the user.